github/actions_workflows.go

```diff
@@ -149,13 +149,26 @@ func (s *ActionsService) getWorkflowUsage(ctx context.Context, url string) (*Wor
 	return workflowUsage, resp, nil
 }
 
-// CreateWorkflowDispatchEvent manually triggers a GitHub Actions workflow run.
+// CreateWorkflowDispatchEventByID manually triggers a GitHub Actions workflow run.
 //
-// GitHub API docs: https://docs.github.com/en/rest/reference/actions#create-a-workflow-dispatch-event
-func (s *ActionsService) CreateWorkflowDispatchEvent(ctx context.Context, owner, repo string, workflowID int64, event CreateWorkflowDispatchEventRequest) (*Response, error) {
+// GitHub API docs: https://docs.github.com/en/free-pro-team@latest/rest/reference/actions#create-a-workflow-dispatch-event
```

Comment:
@n1lesh - can you please explain to me what this extra `free-pro-team@latest` is and why you are adding these to the URL?

I have a utility in this repo that updates URLs, and it's not going to find these.

Thank you!

Reply:
For me, the docs open with `free-pro-team@latest` in the url, even when I open the old url. Maybe they changed it, or it's just for me?

Reply:
Ah!  It does for me now too... it looks like I'll have to update all the URLs again at some point.  :joy:
Thanks, @n1lesh !